opencsp/app/sofast/lib/DisplayShape.py

```diff
@@ -1,77 +1,63 @@
 import numpy as np
 from scipy.interpolate import LinearNDInterpolator
-from scipy.spatial.transform import Rotation
 
 from opencsp.common.lib.geometry.Vxy import Vxy
 from opencsp.common.lib.geometry.Vxyz import Vxyz
-import opencsp.common.lib.tool.hdf5_tools as hdf5_tools
+import opencsp.common.lib.tool.hdf5_tools as ht

```

Comment:
We should probably converge on how we import this library.  I've been using "import ... as h5", but I'm also good with "ht".  I think that "h5" is more descriptive, but "ht" is more consistent with the rest of the code.  Thoughts?

Reply:
Good idea on being consistent. Since you're already using h5, let's stick with that.

Reply:
Updated to ... as h5 in all (two) files.